archetype/config/env/production.js

```diff
@@ -1,37 +1,3 @@
-'use strict'
-
-const winston = require('winston')
-
 module.exports = {
 
-  trailpack: {
```

Comment:
why removing this ? I think it's a good thing to provide basic stuff for prod environment